### ISSUE TITLE
Parenthesized context managers raise E999

### ISSUE THREAD

**Issue by ThirVondukr** [charliermarsh/ruff#847]
Ruff version: 0.0.132
Ruff configuration: Default

Ruff reports E999 (Syntax Error) for parenthesized context managers [that were introduced in python 3.10](https://docs.python.org/3/whatsnew/3.10.html#parenthesized-context-managers)

This raises E999
```
test.py:2:15: E999 SyntaxError: invalid syntax. Got unexpected token 'as'
```
```py
with (
    open("") as a,
    open("") as b,
):
    pass
```

However it works fine if you use `\` to split line:
```py
with open("") as a, \
    open("") as b:
    pass
```

**Comment by g-as** [comment]
Related #54, #286

**Comment by charliermarsh** [comment]
👍 Unfortunately a known problem that I need to invest in resolving :) Same for structural pattern matching.

**Comment by ThirVondukr** [comment]
Np, just made an issue so it's documented and is easier to find ❤️

**Comment by charliermarsh** [comment]
Always appreciated!

**Comment by charliermarsh** [comment]
Marking as a duplicate of #54.
